_posts/2020-08-30-commenting-system-blog.md

```diff
@@ -190,35 +193,27 @@ services.AddAWSService<IAmazonDynamoDB>();
 services.AddAWSService<Amazon.S3.IAmazonS3>();
 ```
     
-    Now, that we are connected, Let's read from database
+    Now, that we are connected, let's read from database
 
-    As we don't have anything to read from the database, first, we add a a record in the database manually to test our reading functionality and I am assuming you know how to add a record in the table. When you have written the record
+    As we don't have anything to read from the database, first, we add a a record in the database manually to test our reading functionality and I am assuming you know how to add a record in the table.
 
 ## Getting all Comments from AWS DynamoDB (By Post)
 In this section, we will get all the comments by Post. So, each post displays only comments that related to it
 
-After registering IAmazonDynamoDB in our applicaiton we are now in the position to insert it using Microsoft Dependency injection in our Controller. 
+After registering IAmazonDynamoDB in our application we are now in the position to insert it using Microsoft Dependency injection in our Controller. 
 
 1. Delete the two controllers, `S3ProxyController` and `ValuesController` in the controllers folder
 2. Right click on **Controllers** folder > **New C# Class**
 3. Name the class **CommentsController**
 4. Make sure the class is derived from `ControllerBase`
 5. Add the below line above the `class` keywords
 ```
-[Route("api/[controller]")]
-[ApiController]
-[EnableCors("AllowOrigin")] 
+[Route("api/[controller]")] // Annotates the route where this controller can be accessed
+[ApiController]             // Denotes that this class is a controller
+[EnableCors("AllowOrigin")] // Enables Cross Origin Resources Sharing throughout the controller
 ```
 
-If you get red squiffly lines, in the above three line, import what's needed
-
-**Code Explanation:**
-
-    `[Route("api/[controller]")]` -  Annotate the route where this controller can be accessed
-
-    `[ApiController]` -  Denotes that this class is a controller
-
-    `[EnableCors("AllowOrigin")]` - Enables Cross Origin Resources Sharing throughout the controller
+If you get red squiggly lines, in the above three lines, import what's needed
```

Comment:
Not clear, why is this happening, do this several time locally to find the cause.

Reply:
You mean it should add the imports automatically?

Reply:
Lines such as:  If you get red squiggly lines, in the above three lines, import what's needed. Need more clarity.

Reply:
Done the change please review [here](https://github.com/zainafzal88/blog/pull/4)